po/ja.po

```diff
@@ -10,33 +10,29 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-03-10 10:37+0000\n"
-"PO-Revision-Date: 2020-02-02 08:16+0000\n"
+"POT-Creation-Date: 2020-03-17 16:33+0000\n"
+"PO-Revision-Date: 2020-03-22 12:47+0000\n"
 "Last-Translator: Ludek Janda <ljanda@redhat.com>\n"
 "Language-Team: Japanese <https://translate.fedoraproject.org/projects/"
 "cockpit/master/ja/>\n"
 "Language: ja\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=1; plural=0\n"
-"X-Generator: Weblate 3.10.3\n"
-
-#: pkg/docker/storage.jsx:261
-msgid " (shared with the OS)"
-msgstr " (OS と共有)"
+"Plural-Forms: nplurals=1; plural=0;\n"
```

Comment:
ech...
lets keep this PR open with `needs-work` and I'll take a look at this, as it is getting really annoying.

Reply:
One solution would be (likely) to use `sed -i "s/\(^.Plural-Forms:.*\);\\\n\"$/\1\\\n\"/" *.po` in `po-refresh`. Not ideal, but workaround.

I also want to take a look if we can use http://docs.translatehouse.org/projects/translate-toolkit/en/latest/commands/index.html

Reply:
Some seddery seems ok to me as a first bandaid.